modules/sdk-coin-eth/src/eth.ts

```diff
@@ -2084,25 +2083,6 @@ export class Eth extends BaseCoin {
     return prefix.concat(message);
   }
 
-  /**
-   *  Create the final signed message hash from the combine share
-   * @param combineShare
-   */
-  constructFinalSignedMessageHash(combineShare: string): string {
```

Comment:
Unrecorded breaking change.

Reply:
Is there any benefit for our end users having access to this for their own purposes rather than deleting it?

Reply:
None whatsoever. Our users care about the final message hash, whether it has the extra bit required by the EVM or not (for non EVM chains) should be an implementation detail (as long as final message hash conform to what the user need).